templates/cli/lib/commands/pull.js.twig

```diff
@@ -253,19 +185,21 @@ const pullMessagingTopic = async () => {
 
 pull
     .command("project")
-    .description("Pulling your {{ spec.title|caseUcfirst }} project")
+    .description("Pulling your Appwrite project name")
```

Comment:
Let's stick to `{{ spec.title|caseUcfirst }}`, unless this was recommended by Christy